yarn.lock

```diff
@@ -9,6 +9,22 @@
   dependencies:
     "@babel/runtime" "^7.11.2"
 
+"@adeira/sx-tailwind@^0.8.0":
```

Comment:
This suggests that SX Tailwind was not upgraded everywhere in the Universe to the latest version. Could you check it?

Reply:
Fixed